app/javascript/components/patient/Patient.js

```diff
@@ -38,85 +42,124 @@ class Patient extends React.Component {
             </h4>
           </Col>
-          <Col md="auto">
-            <span className="jursdiction-user-box">
-              <span id="jurisdiction-path">
+          <Col md="auto" className="jursdiction-user-box mr-3">
+            <Row id="jurisdiction-path">
+              <Col>
                 <b>Assigned Jurisdiction:</b> {this.props.jurisdiction_path ? this.props.jurisdiction_path : '--'}
-              </span>
-              <br />
-              <span id="assigned-user">
+              </Col>
+            </Row>
+            <Row id="assigned-user">
+              <Col>
                 <b>Assigned User:</b> {this.props.details.assigned_user ? this.props.details.assigned_user : '--'}
-              </span>
-            </span>
+              </Col>
+            </Row>
           </Col>
         </Row>
         <Row className="pt-4 mx-1 mb-2">
-          <Col id="identification" md="11">
+          <Col id="identification" md="12">
             <Row>
               <Col>
                 <div className="float-left">
                   <h5>
                     <b>IDENTIFICATION</b>
                   </h5>
                 </div>
-                <div className="float-right">
+                <div>
                   {this.props.goto && (
                     <Button variant="link" className="pt-0" onClick={() => this.props.goto(0)}>
-                      <h5>Edit</h5>
+                      <h5>(Edit)</h5>
                     </Button>
                   )}
                 </div>
                 <div className="clearfix"></div>
               </Col>
             </Row>
             <Row>
-              <Col className="text-truncate">
-                <b>DOB:</b> <span>{this.props.details.date_of_birth && `${moment(this.props.details.date_of_birth, 'YYYY-MM-DD').format('MM/DD/YYYY')}`}</span>
-                <br />
-                <b>Age:</b> <span>{`${this.props.details.age ? this.props.details.age : '--'}`}</span>
-                <br />
-                <b>Language:</b> <span>{`${this.props.details.primary_language ? this.props.details.primary_language : '--'}`}</span>
-                <br />
-                <b>State/Local ID:</b> <span>{`${this.props.details.user_defined_id_statelocal ? this.props.details.user_defined_id_statelocal : '--'}`}</span>
-                <br />
-                <b>CDC ID:</b> <span>{`${this.props.details.user_defined_id_cdc ? this.props.details.user_defined_id_cdc : '--'}`}</span>
-                <br />
-                <b>NNDSS ID:</b> <span>{`${this.props.details.user_defined_id_nndss ? this.props.details.user_defined_id_nndss : '--'}`}</span>
+              <Col className="text-truncate" md="auto">
+                <Row>
+                  <Col>
+                    <b>DOB:</b>{' '}
+                    <span>{this.props.details.date_of_birth && `${moment(this.props.details.date_of_birth, 'YYYY-MM-DD').format('MM/DD/YYYY')}`}</span>
```

Comment:
Cant you just do `moment(this.props.details.date_of_birth).format('MM/DD/YYYY')`?

Reply:
fixed